Assets/MRTK/Tests/PlayModeTests/SolverTests.cs

```diff
@@ -1327,6 +1336,7 @@ private Vector3 DetermineHandOriginPositionOffset(HandConstraint.SolverSafeZone
                 case HandConstraint.SolverSafeZone.BelowWrist:
                     return Vector3.up * WristTestActivationPointModifier;
 
+                case HandConstraint.SolverSafeZone.AtopPalm:
```

Comment:
Can you leave a comment for why this is using the same activation point modifier for finger tips?

All of the other ones use a unique constraint (even if they are still referencing arbitrary decimal values)

Reply:
OK!